src/storage/src/hummock/store/version.rs

```diff
@@ -553,11 +553,23 @@ impl HummockVersionReader {
                 .sstable(sstable_info, &mut local_stats)
                 .in_span(Span::enter_with_local_parent("get_sstable"))
                 .await?;
-            if let Some(prefix_hash) = bloom_filter_prefix_hash.as_ref() {
-                if !hit_sstable_bloom_filter(table_holder.value(), *prefix_hash, &mut local_stats) {
-                    continue;
+            let mut hit_bloom_filter = false;
+            for table_id in &sstable_info.table_ids {
```

Comment:
This looks incorrect to me. We only need to check the bloom filter for one table_id (the one specified in ReadOptions), not bloom filters for all table ids. I guess this is why CI fails.

Reply:
Indeed my fault, missed out there is one table_id in `ReadOption`.